codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyHttpEncoder.java

```diff
@@ -121,15 +121,33 @@
 
     private int currentStreamId;
 
+    private boolean validateHeaders;
```

Comment:
  Original Message                                                                                                                                                                                                                                                                                                                                                                                                                                      From: Norman MaurerSent: Friday, September 4, 2015 17:27To: netty/nettyReply To: netty/nettySubject: Re: [netty] Add validateHeaders and headersToLowerCase options for SPDY (#4195)In codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyHttpEncoder.java:
> @@ -121,15 +121,33 @@
>  
>      private int currentStreamId;
>  
> +    private boolean validateHeaders;

final

—Reply to this email directly or view it on GitHub.